### ISSUE TITLE
Parse release name after options

### ISSUE THREAD

**Issue by spawnia** [SamuelGarrattIqa/helm_upgrade_logs#1]
When called with options that take arguments before the release name, this script interprets the value of the option as the release name.

For example, `helm_upgrade_logs --set tag=latest my-release` will fail with the following error:

```
[INFO] Waiting for pod logs: Error from server (BadRequest): Unable to find "/v1, Resource=pods" that match label selector "app.kubernetes.io/instance=tag=latest", field selector "": found '=', expected: ',' or 'end of string'
```

I believe the following line to be the culprit, more solid CLI parsing would be required: https://github.com/SamuelGarrattIqa/helm_upgrade_logs/blob/32dda67168cbbdf581ef3fcaa6263ae5a7a7d9af/exe/helm_upgrade_logs#L10. I believe it is necessary to tell the parser which options have arguments and which do not.

As a workaround, the call can be rewritten to:
- `helm_upgrade_logs --set=tag=latest my-release`
- `helm_upgrade_logs my-release --set tag=latest`

**Comment by SamuelGarratt** [comment]
Yes that's right. I can't do the fix right now to handle options before the release name but will try to get to this within a week's time

**Comment by SamuelGarrattIqa** [comment]
@spawnia have fixed this in version 0.4.0 just released. Let me know if anything else doesn't work

**Comment by spawnia** [comment]
Thank you, your fix worked perfectly for me.
